src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -15,10 +15,15 @@
               {% if form.vars.multipart %} enctype="multipart/form-data"{% endif %}
               method="POST"
               {% if not sonata_config.getOption('html5_validate') %}novalidate="novalidate"{% endif %}
-              {{ stimulus_controller('sonata-confirm-exit') }}
+              {{
+                stimulus_controller('sonata-confirm-exit')|
+                stimulus_controller('sonata-edit')
+              }}
               {{
                 stimulus_action('sonata-confirm-exit', 'confirm', 'beforeunload@window')|
-                stimulus_action('sonata-confirm-exit', 'skip', 'submit')
+                stimulus_action('sonata-confirm-exit', 'skip', 'submit')|
+                stimulus_action('sonata-edit', 'prepareSubmit', 'submit')|
```

Comment:
I don't really like the name of the controller `edit`
There will be confusion with xeditable `editable` controllers